### PR TITLE
Add CI for odf-console repo in red-hat-storage organization

### PR DESCRIPTION
**Pull request by bipuladh**
Signed-off-by: bipuladh <badhikar@redhat.com>

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]
/hold 

LGTM, unhold at will

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: lint
+    commands: yarn run lint
+    container:
+      from: bin
```
Comment (petr-muller) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:15]: You need `binary_build_commands` to build the `bin` image (see https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts)

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+_generated_metadata:
```
Comment (petr-muller) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:16]: This is an invalid key

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: lint
+    commands: yarn run lint
+    container:
+      from: root
```
Comment (petr-muller) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:15]: `root` does not have your source code injected (which is I assume the thing you want to lint)

Comment (bipuladh) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:15]: would src have it?

Comment (petr-muller) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:15]: I can only link to https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts that describes what image contains what.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**